library/src/blas_ex/rocblas_gemm_ex.cpp

```diff
@@ -94,6 +112,16 @@ extern "C" rocblas_status rocblas_gemm_ex(rocblas_handle    handle,
                     alpha_double = *static_cast<const int32_t*>(alpha);
                     beta_double  = *static_cast<const int32_t*>(beta);
                 }
+                else if(compute_type == rocblas_datatype_f32_c)
+                {
+                    alpha_double = to_double(*static_cast<const rocblas_float_complex*>(alpha));
+                    beta_double  = to_double(*static_cast<const rocblas_float_complex*>(beta));
+                }
+                else if(compute_type == rocblas_datatype_f64_c)
+                {
+                    alpha_double = to_double(*static_cast<const rocblas_double_complex*>(alpha));
+                    beta_double  = to_double(*static_cast<const rocblas_double_complex*>(beta));
+                }
```

Comment:
In complex GEMM, alpha and beta are supposed to be complex, most generally speaking.

So they cannot simply be converted to double, and have their imaginary part ignored.